windows.md

```diff
@@ -0,0 +1,119 @@
+# Running/Developing OpenSphere on Windows 10+
+
+The following are instructions for getting OpenSphere building and running on a Windows 10 system using the new Windows Subsystem for Linux (WSL). Note All of the following instructions assume a Windows 10 build of 16215 or later. 
+
+## Installing WSL
+
+See the following instructions for setting up WSL on Windows 10. 
+
+https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+After installing the WSL feature and selecting a distribution, the following OpenSphere project prerequisites need to be installed using the WSL shell environment.
+
+	* [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+	* [NodeJS 8](https://nodejs.org/en/download/package-manager/)
+	* Oracle Java 8 SDK, OpenJDK should also work
+    * [Umbuntu](https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-get-on-ubuntu-16-04)

```

Comment:
Ubuntu?

Reply:
I was suggesting you meant to use `Ubuntu` where you wrote `Umbuntu`.

Reply:
ahh :(  thanks!